flash/image/classification/data.py

```diff
@@ -380,3 +444,236 @@ def show_post_tensor_transform(self, samples: List[Any], running_stage: RunningS
     def show_per_batch_transform(self, batch: List[Any], running_stage):
         win_title: str = f"{running_stage} - show_per_batch_transform"
         self._show_images_and_labels(batch[0], batch[0][DefaultDataKeys.INPUT].shape[0], win_title)
+
+
+class ImageClassificationDataSourceCollection(DataSourceCollection):
+    def __init__(self, **data_source_kwargs):
+        super().__init__(
+            data_sources={
+                DefaultDataSources.FIFTYONE: ImageFiftyOneDataSource(**data_source_kwargs),
+                DefaultDataSources.FILES: ImagePathsDataSource(),
+                DefaultDataSources.FOLDERS: ImagePathsDataSource(),
+                DefaultDataSources.NUMPY: ImageNumpyDataSource(),
+                DefaultDataSources.TENSORS: ImageTensorDataSource(),
+                DefaultDataSources.DATAFRAME: ImageClassificationDataFrameDataSource(),
+                DefaultDataSources.CSV: ImageClassificationDataFrameDataSource(),
+                DefaultDataSources.LABELSTUDIO: LabelStudioImageClassificationDataSource(),
+            },
+            default_data_source=DefaultDataSources.FILES,
+        )
+
+
+class ImageClassificationDataV2(DataModule):
```

Comment:
The naming of these modules is also a source of confusion and doesn't make it clear that they inherit from DataModule. The other classes in this library follow the convention of having the superclass name suffix the derived class. There is an ImageClassificationDataSource and an ImageClassificationPreproces but not an ImageClassificationDataModule.

Reply:
I totally agree, we should make it clear with `ImageClassificationDataModule`.